my_script.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We should probably move this into a folder called `samples` or something similar.